### ISSUE TITLE
[rutracker] (updating) RuTracker authentication failed

### ISSUE THREAD

**Issue by bylampa** [Jackett/Jackett#15373]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
ver. 0.22.56
```

### Description

Authentication error
I can't add an indexer rutracker.org , there is access , the captcha is empty

### Logged Error Messages

```text
Null
```

**Comment by garfield69** [comment]
Yes, failing for me too, I shall investigate.

**Comment by garfield69** [comment]
It looks like there is something broken with any C# indexer that performs a login. I see failures with ncore #15372, rutracker, funfile, greatposterwall and possibly others.
@ilike2burnthing  @mynameisbogdan  seems related to the processing the document or querying the selector in the document causing it to drop into a throw error.

**Comment by garfield69** [comment]
Yes, I shall publish as soon I retest all my failed indexer, just to be absolutely sure its all good. Thanks for the quick fix.

**Comment by garfield69** [comment]
v0.22.58
